### ISSUE TITLE
Force Close Android when processing start

### ISSUE THREAD

**Issue by juanro49** [BOINC/boinc#3698]
**Describe the bug**
Force close when process WU staart

**Steps To Reproduce**
1. Start app
2. Start process tasks

**Expected behavior**
Process tasks without problems

**Screenshots**
<img src="https://i.imgur.com/RmLuPxh.png"/>

**System Information**
 - OS: LineageOS 15.1 (Android 8.1.0) without GAPPS
 - BOINC Version: 7.16.3 (same problem in 7.4.53)

**Additional context**
this is the log
```
2020-05-09 20:30:01.916 1661-13680/? E/NotificationService: No Channel found for pkg=edu.berkeley.boinc, channelId=main-channel, id=1, tag=null, opPkg=edu.berkeley.boinc, callingUid=10214, userId=0, incomingUserId=0, notificationUid=10214, notification=Notification(channel=main-channel pri=1 contentView=null vibrate=null sound=null defaults=0x0 flags=0x0 color=0x00000000 actions=1 vis=PRIVATE)
2020-05-09 20:30:01.921 1661-1696/? E/NotificationService: Suppressing notification from package by user request.
2020-05-09 20:30:01.922 1661-1696/? W/ActivityManager: Error showing notification for service
    java.lang.RuntimeException: invalid channel for service notification: Notification(channel=main-channel pri=1 contentView=null vibrate=null sound=null defaults=0x0 flags=0x40 color=0x00000000 actions=1 vis=PRIVATE)
        at com.android.server.am.ServiceRecord$1.run(ServiceRecord.java:532)
        at android.os.Handler.handleCallback(Handler.java:790)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:164)
        at android.os.HandlerThread.run(HandlerThread.java:65)
        at com.android.server.ServiceThread.run(ServiceThread.java:46)
2020-05-09 20:30:01.924 27833-27833/? D/AndroidRuntime: Shutting down VM
2020-05-09 20:30:01.924 27833-27833/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: edu.berkeley.boinc:remote, PID: 27833
    android.app.RemoteServiceException: Bad notification for startForeground: java.lang.RuntimeException: invalid channel for service notification: Notification(channel=main-channel pri=1 contentView=null vibrate=null sound=null defaults=0x0 flags=0x40 color=0x00000000 actions=1 vis=PRIVATE)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1768)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
2020-05-09 20:30:01.925 1661-1696/? E/NotificationService: Suppressing notification from package by user request.
2020-05-09 20:30:01.930 1661-1697/? I/ActivityManager: Showing crash dialog for package edu.berkeley.boinc u0
```

**Comment by juanro49** [comment]
For the moment, I'm using boinc with termux that it work fine https://github.com/truboxl/boinc-termux-client

**Comment by talregev** [comment]
Are you using android 8.1 or above?
This is android boinc version of 7.16.4 or above?

**Comment by talregev** [comment]
> * OS: LineageOS 15.1 (Android 8.1.0) without GAPPS
> * BOINC Version: 7.16.3 (same problem in 7.4.53)

Can you verify it version 7.16.3?

**Comment by AenBleidd** [comment]
Should be fixed via #3710
